File: src/main/java/no/nav/melosys/eessi/models/sed/nav/EessiSak.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@JsonInclude(JsonInclude.Include.NON_NULL)
@Data
public class EessiSak {

    private String institusjonsnummer;

    private String institusjonsid;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/EessiSak.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class EessiSak(
    var institusjonsnummer: String? = null,
    var institusjonsid: String? = null
)